### PR TITLE
Feature/generate receipt

### LINKED ISSUE #2335: [FEATURE] - 將整理好的一份政治獻金資料轉成虛擬的收入與支出收據
### Summary

寫一個Daemon將整理好的一份政治獻金資料，將這份資料從資料庫取出轉成虛擬的收入與支出收據，轉成 pdf 或是 jpeg，最終要讓AI識別

### Tasks

**Description**:
需要撰寫一個Daemon來處理已經整理好的政治獻金資料，具體要求如下：

1. 從資料庫中提取政治獻金資料。
2. 將資料轉換為收入與支出的虛擬收據。
3. 支持將虛擬收據輸出為 PDF 或 JPEG 格式。
4. 確保生成的文件可以被 AI 系統進行識別和分析。

**Acceptance Criteria**:
- 成功提取資料並生成虛擬收據。
- 支援 PDF 和 JPEG 格式的輸出。
- AI 能夠正確識別生成的收據內容。

### Dependencies

_No response_

### Other Dependencies

_No response_

### Additional Notes

_No response_

### PR DESCRIPTION
**Pull request by TzuHanLiang**
### DEVELOP

- [x] 新增 table: receipt_sync
- [x] 新增 file: generate receipt

### CHECK LIST

- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new recursive: 0
- high risk: 0
- new sql: 5
    - src/receipt_generator.ts
        - ln28
        - ln227
        - ln240
        - ln247
        - ln253
- new loop: 1
    - ln252-277
- new Library: 2
    - @pdf-lib/fontkit
    - pdf-lib
- new Class / Component: 0

### CODE REVIEW

File: prisma/donations.db
Comment (Luphia) [inline prisma/donations.db:1]: - remove db file

File: src/assets/fonts/SourceHanSansTW-Regular.otf
Comment (Luphia) [inline src/assets/fonts/SourceHanSansTW-Regular.otf:1]: - The file name must comply with the naming conventions

File: src/receipt_generator.ts
Comment (Luphia) [inline src/receipt_generator.ts:1]: - The comment format must be followed

**Review by Luphia — CHANGES_REQUESTED** [review]

File: src/receipt_generator.ts
Comment (TzuHanLiang) [reply src/receipt_generator.ts:1]: fixed

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**